recipes/vacuum/meta.yaml

```
@@ -0,0 +1,35 @@
+{% set name = "vacuum" %}
+{% set version = "0.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/daveshanley/vacuum/archive/v{{ version }}.tar.gz
+  sha256: e149492b6f53b4bef90a8b80649a48421ec89d28a5176406e0ea12ff81ea7a7f
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go-nocgo') }}
+    - go-licenses
+
+test:
+  commands:
+    - vacuum --help
+
+about:
+  home: https://github.com/daveshanley/vacuum
+  license: MIT
```

Due to the above license of the third-party
```suggestion
  license: MIT AND BUSL-1.1
```

---

Whoops, I missed that.  See the last commit for the fix for this.